docs/ort_genai_igpu.md

```diff
@@ -0,0 +1,50 @@
+# OnnxRuntime GenAI (OGA) for iGPU and CPU
+
+onnxruntime-genai (aka OGA) is a new framework created by Microsoft for running ONNX LLMs: https://github.com/microsoft/onnxruntime-genai/tree/main?tab=readme-ov-file
+
+## Installation
+
+To install:
+
+1. `conda create -n oga-igpu python=3.9`
+1. `conda activate oga-igpu`
+1. `pip install -e path/to/genai[oga-igpu]`
```

Comment:
This line refers to internal genai repo, should this be `pip install -e .[llm-oga-dml]`

Reply:
Fixed in https://github.com/onnx/turnkeyml/pull/244/commits/f50505a733fb2305890e10df9c2dd72888b0c2c7